src/main/scala/rocket/Frontend.scala

```diff
@@ -59,7 +59,7 @@ class Frontend(val icacheParams: ICacheParams, hartid: Int)(implicit p: Paramete
   val masterNode = TLOutputNode()
   val slaveNode = TLInputNode()
 
-  icache.slaveNode.map { _ := slaveNode }
+  icache.slaveNode.map { _ connectButDontMonitor slaveNode }
```

Comment:
Why is this one no longer being monitored? The other place monitors are disabled has a nice comment.